### PR TITLE
Comments: enable single comment view

### PR DESCRIPTION
**Pull request by rodrigoi**
This PR enables the links to the comment view on:

- The comment timestamp on _Site Comments_ and _Post Comments_ views.
- The comment title on the _In reply to_ label for any comment that is a reply, for WordPress.com sites.

It also enables moderation hotlinks from emails or `wp-admin` (#19774).

How to test:

Both the timestamp below the user name and the title next to the _In reply to_ label should point to the single comment view.

### CODE REVIEW

File: config/development.json [hunk 1]
```diff
@@ -42,7 +42,6 @@
 		"code-splitting": true,
 		"comments/filters-in-posts": true,
 		"comments/moderation-tools-in-posts": true,
-		"comments/management/comment-view": true,
```
Comment (gwwar) [inline config/development.json]: I think it's fine to enable this in other envs first, before removing all config checks. We can do that in a separate PR when things look OK. (Makes it super easy to turn off if we need to).

Comment (vindl) [reply config/development.json]: +1

**Comment by vindl** [comment]
Navigating from timestamps and `in reply to` tests well for me. One thing that I've noticed is that the notices are no longer displayed for single comment actions (via `action` parameter) and I vaguely remember seeing them before. If that's expected though feel free to :shipit:.

**Comment by Copons** [comment]
@vindl That's probably to be expected, since the `action` param handler doesn't show any notices: https://github.com/Automattic/wp-calypso/blob/master/client/components/data/moderate-comment/index.jsx#L50-L64

I don't remember if we discussed this, but in case, it should be as simple as copying this method (maybe dropping the undo logic): https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/comments/comment/comment-actions.jsx#L92-L115

**Comment by rodrigoi** [comment]
@vindl you are absolutely right, there use to be success notifications. Haven't tracked what changed in the month that this was hidden under the feature flag, but I've implemented @Copons suggestions to restore them.
@gwwar I've also reverted the changes that removed the feature flag and added it to all environments.

**Comment by gwwar** [comment]
👍  retested and the notices appear correctly! Feel free to 🚢  when ready
